docs/1.16/prisma-client/api-reference/schema-delegation-GO-pyl9.mdx

```diff
@@ -0,0 +1,11 @@
+import Collapse from 'components/Markdown/Collapse'
+
+export const meta = {
+  title: 'Schema Delegation (Go)',
+  position: 80,
+  technology: "go",
+  technologyOrder: 2,
+  articleGroup: "Schema Delegation",
+}
+
+Coming Soon
```

Comment:
do we have a feature request we can link here? if no, can you maybe create a simple one?